### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by nielsdrost** [open-hydrology/open-hydrology.github.io#2] [General]
Welcome to the Open Hydrology Discussions!

If you have a comment, question, or anything else you would like to share with the community feel free to start a new discussion.

**Comment by javedali99** [comment]
This [list of python resources for hydrology](https://github.com/javedali99/python-resources-for-earth-sciences) might be useful for others. Please add it on the resources pages, if you find it useful.

**Reply by sheilasaia** [reply]
Thanks so much for this suggestion, @javedali99. 🙌  I'm adding it to our v2 discussion (here: #5). Please reach out if you find additional resources that we can add.
